### PR TITLE
Cleanup composer.json

### PR DESCRIPTION
**Pull request by MattKetmo**
- Move from `~` to `^`
- Replace `|` by `||`
- Use PSR-4

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -11,27 +11,27 @@
...
+        "php": "^5.4 || ^7.0",
+        "symfony/options-resolver": "^2.3 || ^3.0",
+        "psr/log": "^1.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "~4.7|~5.0",
-        "php-amqplib/php-amqplib": "~2.1",
-        "doctrine/common": "~2.3",
-        "doctrine/dbal": "~2.2"
+        "phpunit/phpunit": "^4.7 || ^5.0",
```
Comment (Taluu) [inline composer.json]: I wonder if we should not remove the phpunit dependency, and use the global one on travis instead ? Or even suggest using the global one if we need to run the tests...

Comment (MattKetmo) [inline composer.json]: Travis tests already uses its global one.
But personally I don't have phpunit installed globally anymore :)

Comment (odolbeau) [inline composer.json]: Same for me. ;)

Comment (Taluu) [inline composer.json]: I actually did the reverse, I installed it globally not too long ago. :D

Comment (MattKetmo) [inline composer.json]: now you can run `composer test` instead of `phpunit` ;)

**Comment by odolbeau** [comment]
Thanks a lot! <3

**Pull request merged**